system/modules/core/dca/tl_files.php

```diff
@@ -146,7 +146,8 @@
 		),
 		'path' => array
 		(
-			'sql'                     => "varchar(1022) NOT NULL default ''"
+			'eval'                    => array('unique'=>true),
```

Comment:
??? That's what the whole PR and discussion today was about?? 20x `FilesModel::findByPath('path.jpg')` should return the same instance because it's unique? If not, what combination of columns does make it unique then?